Source/movie.cpp

```diff
@@ -23,6 +23,11 @@ void play_movie(const char *pszMovie, BOOL user_can_close)
 {
 	HANDLE video_stream;
 
+	int cursor_shown = SDL_ShowCursor(SDL_DISABLE);
+	if (cursor_shown <= -1) {
+		SDL_Log(SDL_GetError());
+	}

```

Comment:
Since the cursor is now drawn by the os we need to inform it that we don't want it displayed during video playback. For the in engine cursor the engine was it self aware when video was being played and wouldn't display the cursor if that was the case.

Reply:
What @AJenbo said, basically.